packages/components/src/DataList/components/DataListHeaderTile/DataListHeaderTile.css

```diff
@@ -0,0 +1,18 @@
+.headerLabel {
+  display: flex;
+  align-items: center;
+}
+
+button.headerLabel {
```

Comment:
i think we can just put this on the other one too. No need to differentiate between them as they should have the same styling regardless